app/models/patient.rb

```diff
@@ -10,6 +10,23 @@ class Patient < ApplicationRecord
   include ActiveModel::Validations
   include FhirHelper
 
+  MONITORING_REASONS = [
```

Comment:
Since this is already defined in `ValidationHelper` at `VALID_PATIENT_ENUMS[:monitoring_reason]`, let's just use that instead of adding more to this model. Look at how `monitoring_reason` is validated a bit below this. If you don't want to pass the empty string or nil option to the front end, you can just filter those out with a `.reject(&:blank?)`.

Reply:
The validation helper includes extra options that we dont want to present to the user. For example, the `(system)` options, like `Enrolled on last day of monitoring period (system)`, which the user should not be able to select.

Reply:
Then perhaps ValidationHelper need to incorporate this array when building its own list (with the additional few we don't want the front end to see). Without combining these two things in some way, all this PR is doing is moving the duplicated list from the frontend to the backend. That's really not consolidating much of anything.

Reply:
Resoved in latest commit.